agriMushroom/ngsild-payloads/agriParcel.json

```diff
@@ -11,11 +11,11 @@
             ]
         }
     },
-    "cropStatus": {
+    "https://uri.fiware.org/ns/data-models#cropStatus": {

```

Comment:
I think this could only be seen when getting the entity and maybe is related to a specific Context Broker.